script/testing/oltpbench/constants.py

```diff
@@ -37,3 +44,11 @@
     OLTP_ANT_COMMAND_BOOTSTRAP, OLTP_ANT_COMMAND_RESOLVE,
     OLTP_ANT_COMMAND_BUILD
 ]
+
+# Performance Storage Service
```

Comment:
Add documentation to explain what these are.

Reply:
Done